recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+proj:
+    - '7.2.0'
```

Comment:
Proj is pinned down to a micro version so we need to be consistent with the previous pin, 7.2.0, even though we packaged 7.2.1.

Reply:
Will the 8.0.1 automatically be updated? Or does this mean we now need to manually update this?

Reply:
>8.0.1 will be added via migrations but the solution here should be "temporary" and we should remove this file as soon as cartopy works with latest proj.

Reply:
👍